app/views/handlers/question.py

```diff
@@ -116,7 +121,7 @@ def get_next_location_url(self):
             self._return_to_block_id,
         )
 
-    def _get_answers_for_question(self, question_json) -> dict[str, Any]:
+    def _get_answers_for_question(self, question_json: dict) -> dict:
```

Comment:
Arguments should be abstract. Unless this needs dict specific methods, use `Mapping`.